server/aws-lsp-codewhisperer/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [0.0.12] - 2024-08-19
+- Allow sending document without active focus in Chat requests
+- Implement sample device auth SSO flow in LSP server
```

Comment:
this was done in an unrelated server. Shouldn't be in the codewhisperer changelog